.github/workflows/dashboards-issue-add-label.yml

```
@@ -0,0 +1,65 @@
+name: When an issue changes and it's part of the dashboards project, add the dashboards squad label
+on:
+  issues:
+    types: [opened, closed, edited, reopened, assigned, unassigned, labeled, unlabeled]
+      
+env:
+  GITHUB_TOKEN: ${{ secrets.ISSUE_COMMANDS_TOKEN }}
+  ORGANIZATION: ${{ github.repository_owner }}
+  REPO: ${{ github.event.repository.name }}
+  TARGET_PROJECT: 202
+  LABEL_IDs: "LA_kwDOBwinTs8AAAABURA8vg"
```

```suggestion
  LABEL_IDs: "LA_kwDOAOaWjc8AAAABT38U-A"
```
This is the id of `team/grafana-dashboards` in this repo